api/repositories/org_repository.go

```diff
@@ -393,6 +401,91 @@ func (r *OrgRepo) GetSpace(ctx context.Context, info authorization.Info, spaceGU
 	return spaceRecords[0], nil
 }
 
+func (r *OrgRepo) DeleteOrg(ctx context.Context, info authorization.Info, message DeleteOrgMessage) error {
+	// Fetch 'hierarchy' object using the privileged client as we want to return a 'not-found' error for
+	// non-existent orgs as opposed to saying 'permission-denied'.
```

Comment:
I'm not sure if you got notifications for the comment I deleted, but this comment has sent me on a journey. I'd suggest just deleting the comment, since the test suite documents the behavior and the comment doesn't make sense until you've read the whole method.

Reply:
Addressed in commit - https://github.com/cloudfoundry/cf-k8s-controllers/pull/464/commits/a99f88f188236f227f4a094857f8ada66c9204e7